server/src/main/java/org/elasticsearch/bootstrap/BootstrapChecks.java

```diff
@@ -718,5 +784,10 @@ public BootstrapCheckResult check(BootstrapContext context) {
         ByteOrder nativeByteOrder() {
             return ByteOrder.nativeOrder();
         }
+
+        @Override
+        public ReferenceDocs referenceDocs() {
+            return null;
```

Comment:
Hmm do we not have docs for this check? That's a bug if so. Could you use the root `ReferenceDocs.BOOTSTRAP_CHECKS` for now to avoid a `null`, we'll update this later when fixing the docs.